app/helpers/course_summaries_helper.rb

```diff
@@ -0,0 +1,70 @@
+module CourseSummariesHelper
+
+	###
+	# Get JSON data for the table in the form of
+	# { id: student_id, user_name: student.user_name,
+	#   first_name: student.first_name, last_name: student.last_name
+	#   marks: [ // an array of marks for each assignment, null if no grade ] }
+	###
+	def get_table_json_data()
```

Comment:
Omit the parentheses in defs when the method doesn't accept any arguments.<br>Use 2 (not 1) spaces for indentation.<br>Tab detected.